ui/src/Modals/NewLibrary/Index.jsx

```diff
@@ -105,6 +127,17 @@ function NewLibraryModal(props) {
             <div className="separator" />
           </div>
           <div className="fields">
+            {serverError && serverError.message && (
+              <div className="serverError">{serverError.message}</div>
+            )}
+            {serverError &&
+              serverError.payload &&
+              serverError.payload.duplicates &&
+              serverError.payload.duplicates.length > 0 && (
+                <div className="serverErrorDuplicates">
+                  {serverError.payload.duplicates}
+                </div>
+              )}
```

Comment:
I'm not a big fan of this. I'm not entirely sure how this is gonna look UX wise. I haven't thought about how we want to display errors like this to the user yet.

Reply:
Yes, it's not the nicest thing, as it shows as a javascript array. So probably iterating over the list and turn it into a <ul/><li/>.. what ever, would be a better solution.

Reply:
I think for now I will avoid displaying the error in the front end. I'll talk to our designer and see if we can come up with something.